src/rules/labelPositionRule.ts

```diff
@@ -29,10 +29,10 @@ class LabelPositionWalker extends Lint.RuleWalker {
     public visitLabeledStatement(node: ts.LabeledStatement) {
         const statement = node.statement;
         if (statement.kind !== ts.SyntaxKind.DoStatement
-                && statement.kind !== ts.SyntaxKind.ForStatement
-                && statement.kind !== ts.SyntaxKind.ForInStatement
-                && statement.kind !== ts.SyntaxKind.WhileStatement
-                && statement.kind !== ts.SyntaxKind.SwitchStatement) {
+            && statement.kind !== ts.SyntaxKind.ForStatement
+            && statement.kind !== ts.SyntaxKind.ForInStatement
+            && statement.kind !== ts.SyntaxKind.WhileStatement
+            && statement.kind !== ts.SyntaxKind.SwitchStatement) {
```

Comment:
good point, that's always a possibility for any massive conditional, or it could be a variable beforehand as well.